src/connection/arrayconnection.js

```diff
@@ -122,6 +122,7 @@ export function connectionFromArraySlice<T>(
   var upperBound = before ? beforeOffset : arrayLength;
   return {
     edges,
+    totalCount: arrayLength,
```

Comment:
This may work for arrays, but not necessarily for all kinds of connections.

Is there a variant of this which adds this field only for arrayconnection but leaves it up to you (via `connectionFields`) to add otherwise?